### ISSUE TITLE
[Feature Request] Add applications to ThreadFix via BoH

### ISSUE THREAD

**Issue by tahersb** [PearsonEducation/bag-of-holding#2]
Would be nice to use BoH to create/maintain applications inventory and have those applications created/added in ThreadFix. 

Currently BoH requires an application to already exist in ThreadFix first before it can establish the link. While this is a good feature when you already have existing applications in ThreadFix, it would also be nice to enable BoH to create applications in ThreadFix.

**Comment by aparsons** [comment]
Makes sense. When I rework how threadfix connects to applications, I'll add an option to create a new application in threadfix.

Thanks for the suggestion!

**Comment by NoahJaehnert** [comment]
I second this request. Would like to have this, so I don't have to duplicate work!

Speaking of this specific request -- In one of Matt Tesauro's presentations, he discusses automating onboarding of apps in Threadfix, Bag of Holding, and $ScanTool. I tweeted to him asking for more details but didn't hear anything back yet sadly (https://twitter.com/njaehner/status/667395609209520128)

Thanks!

-Noah

**Comment by aparsons** [comment]
I'll ping Matt and let him know.

**Comment by NoahJaehnert** [comment]
Matt responded on Twitter, FYI. Wanted to include here as reference for others: 

> @njaehner We used StackStorm + some Python glue code to do that work.  I mentioned our @Checkmarx integration in my talk. [1/2] 

(From: https://twitter.com/matt_tesauro/status/672128964484182016)

> @njaehner Our chatbot talks to @Stack_Storm which then has the code to talk to Bag of Holding, @ThreadFix and @Checkmarx [2/2] 

(From: https://twitter.com/matt_tesauro/status/672129416663719937)
